### ISSUE TITLE
Auto-turn on feedback after assignment is due

### ISSUE THREAD

**Issue by cbmajor7** [artusiinc/issues#59]
Is it possible to include a box that automatically turns on instructor feedback after an assignment is due? Alternatively (or additionally), it might be nice to allow students to 'submit' an assignment, after which they're locked out of it, and then receive feedback on submitting.

**Comment by mscuthbert** [comment]
Hi Josh -- these are both great suggestions.  In fact, we have the first one in the code and at one point (a long, long time ago) it was turned on, but then there were issues about students using makeup assignments being able to see exams, professors who like to be a bit lenient about deadlines, etc.  I'll make a Course policies on deadlines setup page and then professors can configure their own.

Limiting number of submissions / number of checks is on the roadmap as an option as well.  It's not generally something I do in my own teaching, but there are solid pedagogical reasons for doing it.

**Comment by cbmajor7** [comment]
Awesome! Thanks!

Does this mean that the new version has been released?

**Comment by cbmajor7** [comment]
Cool! You mentioned a new bigger update yesterday that required testing on
minor browsers. Is that out yet? In general, are there notification for
when new versions become available?

**Comment by mscuthbert** [comment]
Yep! it's out!  Sorry, should've flagged you on that.  At present we haven't been announcing new releases -- most of them are too minor to really care about (yesterday's included "adjusted the size of pitch-identification input box to be a bit bigger" :-)  ) but I'll bring to the logistics team attention.
